# Description: disable _nss_dns_gethostbyname4_r for the moment, as it
#  causes problems for IPv6 users; patch from Fedora by Jakub Jelinek
# Ubuntu: https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/313218
# Upstream: https://bugzilla.redhat.com/show_bug.cgi?id=459756

Index: b/resolv/Versions
===================================================================
--- a/resolv/Versions
+++ b/resolv/Versions
@@ -102,7 +102,7 @@
     _nss_dns_gethostbyname_r; _nss_dns_getnetbyaddr_r;
     _nss_dns_getnetbyname_r; _nss_dns_getcanonname_r;
     _nss_dns_gethostbyaddr2_r;
-    _nss_dns_gethostbyname4_r;
+#    _nss_dns_gethostbyname4_r;
   }
 }
 
